### PR TITLE
Updated i18n-tool version, .travis.yml and Makefile files changed

### PR DESCRIPTION
**Pull request by iivic**
@clintonb

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=167&repo=edx%2Fprograms

**Comment by iivic** [comment]
Please review latest changes. @clintonb

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,9 @@ before_install:
   - "sh -e /etc/init.d/xvfb start"
 install:
   - pip install -U pip wheel codecov
-  - pip install -r requirements/test.txt
+  - make requirements
 script:
+  - make detect_changed_source_translations
```
Comment (clintonb) [inline .travis.yml]: `make validate_translations`

**Comment by iivic** [comment]
Comment addressed.

**Pull request merged**